src/index.jsx

```diff
@@ -118,95 +135,61 @@ export default class UIIdentificationCard extends React.Component {
   }
 
   /**
-   * @getter {String} path
-   */
-  get path() {
-    return [
-      'M165.845472,0',
-      'L227.99565,0',
-      'C232.416331,0',
-      '236,3.57268443',
-      '236,7.99770351',
-      'L236,141.002296',
-      'C236,145.419306',
-      '232.410871,149',
-      '227.99565,149',
-      'L74.7409935,149',
-      'C81.7669784,130.92177',
-      '98.4541553,89.6860279',
-      '117.435739,54.7599123',
-      'C134.773927,22.8576428',
-      '154.983589,6.85502733',
-      '165.845472,-4.88498131e-15 Z',
-    ].join(' ');
-  }
-
-  /**
    * @constructor
    */
   constructor(props) {
     super(props);
     this.state = {
       fillColor: props.fillColor,
       fillOpacity: props.fillOpacity,
-      contentPosition: 0,
+      shadow: 'none',
+      backgroundPosition: '0px 0px'
```

Comment:
Could you make the above 2 states into a pure number, and compute their values when rendering, thank you :-)